.github/ISSUE_TEMPLATE/app_rendering_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Application Card Rendering Bug Report
+about: Create a bug report for problems rendering Adaptive Cards in an Application
+title: "[Rendering] Bug Title Here"
+labels: Bug Triage-Needed Rendering
+assignees: ''
+---
+
+# Target Platform
+
+Please list the platform(s) that your application is running on (ex: Web, iOS, Android, etc)
```

Comment:
It was nice having the pre-populated list of platforms that we can just delete as needed.

Reply:
The problem with the lists is that it doesn't track who toggles them / when they are toggled...this makes it very easy to lose the initial context of what the initial issue state was. It's very easy to accidentally toggle a check-box and not even know it...and once it is toggled, there's no tracing what the initial state was

Reply:
Agreed - I don't think the checklists are necessary, especially now that we have subtasks in ZenHub, A simple bulleted list is what I had in mind.

Reply:
Sounds good, I'll add a list and change the text to ask folks to remove platforms they are not targeting